components/ml/org.wso2.carbon.ml.core/src/main/java/org/wso2/carbon/ml/core/impl/MLModelHandler.java

```diff
@@ -864,6 +909,13 @@ public String publishModel(int tenantId, String userName, long modelId, Format m
                     throw new MLModelPublisherException(errorMsg, e);
                 } catch (MLPmmlExportException e) {
                     throw new MLPmmlExportException("PMML export not supported for model type");
+                } finally {
+                    if (in != null) {
+                        try {
+                            in.close();
+                        } catch (IOException ignore) {
+                        }
```

Comment:
logged ioExceptions without ignoring. Fixed in all places with PR https://github.com/wso2/carbon-ml/pull/202